fava/default-settings.conf

```diff
@@ -28,6 +28,8 @@ journal-show-type-note = True
 journal-show-type-document = True
 journal-show-type-pad = False
 journal-show-type-query = False
+journal-show-type-custom = True
+journal-show-type-budget = True
```

Comment:
If custom directives "become a thing" and people start using them, along with SQL, I think it would make sense to have a custom toggle buttom for them. 

Same goes for budgets: If this feature becomes more mature and maybe there is an official "budget" directive, it's own button makes sense as well, IMHO.